_episodes/dataflow.md

```diff
@@ -17,3 +17,192 @@ learners should be able to
 __explain how a workflow document describes the input and output of a workflow and the flow of data between tools__
 
 {% include links.md %}
+
+
+
+CWL Workflows are about chaining different steps.
+
+A workflow is the orchestration of the individual steps.
+
+
+
+## Definitions
+
+> A CWL Tool description is a document that enables to run a command line tool, using a CWL runner (i.e. any type of workflow engine that is CWL compatible)
+
+> A CWL Workflow description is a particular chaining of steps (running CWL Tools or CWL Workflows) that enables a their execution with a CWL runner in an orchestrated way.
+
+
+This is what you see in a header of a CWL Tool Description
+
+(link: https://github.com/common-workflow-library/bio-cwl-tools/blob/release/bwa/BWA-Index.cwl)
+
+```
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+requirements:
+  DockerRequirement:
+    dockerPull: "quay.io/biocontainers/bwa:0.7.17--ha92aebf_3"
+  InlineJavascriptRequirement: {}
+
+inputs:
+  InputFile:
+    type: File
+    format: edam:format_1929  # FASTA
```

Comment:
For the first time seeing a workflow, I am not sure we need to have the format statements, etc.   I , personally, think simpler is better when people are seeing a workflow for the first time.  Since this is focused on workflow thinking, we want to them to understand the concept of the workflow and how the pieces/data connect.  I know this is best practices, but it might be better served elsewhere.